docs/services/motion/constraints.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Configure Motion Constraints"
+linkTitle: "Constraints"
+weight: 10
+type: "docs"
+description: "Configure constraints to specify certain types of motion."
+---
+
+You can control the motion of your robot using the Motion Service's built-in constraint options.
+
+Available constraints all control the topological movement of the moving component along its path.

```

Comment:
except if there's already a collision, right?

Reply:
Not sure I understand

Reply:
@spieswl can I just delete this sentence?

Reply:
I'd nuke it. It's just filler.